workflows/maskrcnn-training/template.yaml

```diff
@@ -41,25 +41,26 @@ templates:
     tasks:
     - name: train-model
       template: tensorflow
-# Uncomment the lines below if you want to send Slack notifications
-#    - arguments:
-#        artifacts:
-#        - from: '{{tasks.train-model.outputs.artifacts.sys-metrics}}'
-#          name: metrics
-#        parameters:
-#        - name: status
-#          value: '{{tasks.train-model.status}}'
-#      dependencies:
-#      - train-model
-#      name: notify-in-slack
-#      template: slack-notify-success
+    # Uncomment following lines to get Slack notifications
```

Comment:
This should be commented with the same indentation as before.

Reply:
ok, updated.